### PR TITLE
Analytics 7.11.0

### PR DESCRIPTION
**Pull request by moemcd**
Analytics 7.11.0 podspec.json files, SPM, and Firebase podspec updates.

### CODE REVIEW

File: Firebase.podspec [hunk 1]
```diff
@@ -62,6 +62,19 @@ Simplify your app development, grow your user base, and monetize more effectivel
     ss.dependency 'Firebase/Core'
   end
 
+  s.subspec 'CoreWithoutAdIdSupport' do |ss|
```
Comment (paulb777) [inline Firebase.podspec:65]: I don't think there's a reason for this subspec.

Instead:

 ```
 s.subspec 'AnalyticsWithoutAdIdSupport' do |ss|
    ss.ios.deployment_target = '9.0'
    ss.ios.dependency 'FirebaseAnalytics/WithoutAdIdSupport', '~> 7.11.0'
    ss.dependency 'Firebase/CoreOnly'
  end
```

File: Firebase.podspec [hunk 2]
```diff
@@ -62,6 +62,19 @@ Simplify your app development, grow your user base, and monetize more effectivel
...
+    ss.ios.dependency 'FirebaseAnalytics/WithoutAdIdSupport', '~> 7.11.0'
+    ss.dependency 'Firebase/CoreOnly'
+  end  
+
+  s.subspec 'AnalyticsWithoutAdIdSupport' do |ss|
+    ss.ios.deployment_target = '9.0'
+    ss.dependency 'Firebase/CoreWithoutAdIdSupport'
+  end
+
   s.subspec 'ABTesting' do |ss|
```
Comment (paulb777) [inline Firebase.podspec:78]: Consider adding an AnalyticsWithAdIdSupport subspec that is a currently synonym for the Analytics subspec to support forward migration if we go to Phase 3.

File: FirebaseAnalyticsWithoutAdIdSupportWrapper/dummy.m [hunk 3]
```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2020 Google LLC
```
Comment (paulb777) [inline FirebaseAnalyticsWithoutAdIdSupportWrapper/dummy.m:2]: 2021

File: FirebaseAnalyticsWithoutAdIdSupportWrapper/include/dummy.h [hunk 4]
```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2020 Google LLC
```
Comment (paulb777) [inline FirebaseAnalyticsWithoutAdIdSupportWrapper/include/dummy.h:2]: 2021

**Review by paulb777 — COMMENTED** [review]
Thanks for putting this together.

Please fix the CI failures.

**Comment by ryanwilson** [comment]
> Please fix the CI failures.

On the new line 122 of Package.swift, the version dependency needs to change from `7.10.0` to `7.11.0` to point to the new repo.

**Comment by ryanwilson** [comment]
New URL for the 7.11.0 binary is: 
`https://dl.google.com/firebase/ios/swiftpm/7.11.0/FirebaseAnalytics.zip`

New checksum:
`96fdd7d7e7812748b11ec2e8ca9c0cec186e1caa6037996b74410d89a4ddf6af`

**Review by ryanwilson — APPROVED** [review]
LGTM on green CI, thanks!

**Pull request merged**